api/managementpb/dbaas/kubernetes.proto

```diff
@@ -19,11 +19,17 @@ message KubeAuth {
   ];
 }
 
+// KubernetesCluster contains public info about kubernetes cluster.
+message KubernetesCluster {
```

Comment:
Then we need to use it in RegisterKubernetesClusterRequest too

Reply:
In RegisterKubernetesClusterRequest might be more fields, not only public info. We used different messages in other endpoints.